cmd/nox/document.go

```diff
@@ -102,7 +102,7 @@ setting up mappings, shard counts, replicas, etc.`,
 func init() {
 	rootCmd.AddCommand(documentCMD)
 	documentCMD.AddCommand(documentGet, documentIndex, documentDelete,
-		documentDeleteByQuery, documentMultiGet, documentReindex, documentDelete, documentUpdate)
+		documentDeleteByQuery, documentMultiGet, documentReindex, documentUpdate)
```

Comment:
This is the actual relevant change, the rest is just noise from regenerating the documentation

Reply:
🦅 👁️